packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -0,0 +1,44 @@
+import { ContextualMenuItemProps, ContextualMenuProps } from '@fluentui-react-native/contextual-menu';
+import { IButtonProps } from '@fluentui-react-native/button';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { IconProps } from '@fluentui-react-native/icon';
+
+export const MenuButtonName = 'MenuButton';
+
+export interface MenuButtonContext {
+  showContextualMenu?: boolean;
+}
+
+export interface MenuButtonState {
+  context: MenuButtonContext;
+}
+
+export interface MenuButtonItemProps extends ContextualMenuItemProps {
+  submenu?: boolean;
+  submenuItems?: ContextualMenuItemProps[];
+}
+
+export interface MenuButtonProps {
+  menuItems?: MenuButtonItemProps[];
+  content?: string;
+  onItemClick?: (key: string) => void;
+  disabled?: boolean;
+  icon?: number | string | IconProps;
+  contextualMenu?: ContextualMenuProps;
+}
```

Comment:
export interface MenuButtonProps extends IButtonProps {
  menuItems?: MenuButtonItemProps[];
  contextualMenu?: ContextualMenuProps;
}

We should extend IButtonProps to inherit all the acessibility props and pressable props.

Reply:
Yeah, good catch, thank you :)